### ISSUE TITLE
.k-grid-content>table element must have role=grid

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#7204]
### Bug report

.k-grid-content>table element must have role=grid

The role has been removed in the 2023 R1 2023.1.117 release

**Comment by veselints** [comment]
The WAI-ARIA definition for the `grid` role is:
`A composite [widget](https://www.w3.org/TR/wai-aria-1.2/#widget) containing a collection of one or more rows with one or more cells where some or all cells in the grid are focusable by using methods of two-dimensional navigation, such as directional arrow keys.`
https://www.w3.org/TR/wai-aria-1.2/#grid

Hence, the `grid` role should be used only when navigation is enabled in the component (focusing cells/rows). When navigation is disabled the default semantics of the `<table>` element must be used (no explicit role should be added).
